scaling_performance/optimizing_on_glusterfs_storage.adoc

```diff
@@ -0,0 +1,83 @@
+[[scaling-performance-optimizing-on-glusterfs-storage]]
+= Optimizing On GlusterFS Storage
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+toc::[]
+include::_snippets/glusterfs.adoc[]
+
+[[{gluster-native}-Guidance-For-Databases]]
```

Comment:
This instance won't work, since it expands to a string with whitespace. just `Guidance-For-Databases` should be fine.